src/alembic/versions/75857bf0c826_introduce_officer_tables.py

```diff
@@ -0,0 +1,108 @@
+"""introduce officer tables
+
+Revision ID: 75857bf0c826
+Revises: 0db2c57ce969
+Create Date: 2024-06-03 06:36:40.642476
+
+"""
+
+from collections.abc import Sequence
+from typing import Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "75857bf0c826"
+down_revision: str | None = "0db2c57ce969"
+branch_labels: str | Sequence[str] | None = None
+depends_on: str | Sequence[str] | None = None
+
+
+def upgrade() -> None:
+    # drop all existing user session data
+    # TODO: combine all past migrations into this one
+    op.drop_table("user_session")
+    op.create_table(
+        "user_session",
+        sa.Column("session_id", sa.String(512), nullable=False, primary_key=True),
+        sa.Column("issue_time", sa.DateTime, nullable=False),
+        sa.Column("computing_id", sa.String(32), nullable=False),
+    )
+    op.create_unique_constraint("unique__user_session__session_id", "user_session", ["session_id"])
+    op.create_unique_constraint("unique__user_session__computing_id", "user_session", ["computing_id"])
+
+    # drop all existing site user data
+    # TODO: combine all past migrations into this one
+    op.drop_table("site_user")
+    op.create_table(
+        "site_user",
+        sa.Column("computing_id", sa.String(32), nullable=False, primary_key=True),
+    )
+    op.create_unique_constraint("unique__site_user__computing_id", "site_user", ["computing_id"])
```

Comment:
A primary key must already be unique, so this constraint is redundant

Reply:
Also, HECK YES to computing_id being the primary key!! 🥳

Reply:
very cool idea, will update it!

also; if we find a duplicate computing_id we can call sfu it and complain \:)

Reply:
agreed!

Reply:
https://github.com/CSSS/csss-site-backend/pull/26/commits/5b3a3a8e41149355d1b16c6c3375049759199b1c

sorry for the delay! (sob)